src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props

```diff
@@ -261,13 +256,13 @@
     <RuntimeGroup Include="ubuntu">
       <Parent>debian</Parent>
       <Architectures>x64;x86;arm</Architectures>
-      <Versions>14.04;14.10;15.04;15.10</Versions>
+      <Versions>14.04</Versions>
       <TreatVersionsAsCompatible>false</TreatVersionsAsCompatible>
     </RuntimeGroup>
     <RuntimeGroup Include="ubuntu">
       <Parent>debian</Parent>
       <Architectures>x64;x86;arm;arm64</Architectures>
-      <Versions>16.04;16.10;17.04;17.10;18.04;18.10;19.04;19.10;20.04;20.10;21.04;21.10;22.04</Versions>
+      <Versions>16.04;18.04;20.04;22.04</Versions>
```

Comment:
Removed versions are EOL 2022-07-14 and before (https://wiki.ubuntu.com/Releases).